art/estimators/gan/tensorflow.py

```diff
@@ -42,29 +43,31 @@ def __init__(
         discriminator_optimizer_fct=None,
     ):
         """
-        Initialization of a test TF2 GAN
+        Initialization of a test TensorFlow v2 GAN
+
         :param generator: a TensorFlow2 generator
-        :param discriminator: a TensorFlow 2 discriminator
+        :param discriminator: a TensorFlow v2 discriminator
         :param generator_loss: the loss function to use for the generator
         :param discriminator_loss: the loss function to use for the discriminator
         :param generator_optimizer_fct: the optimizer function to use for the generator
         :param discriminator_optimizer_fct: the optimizer function to use for the discriminator
         """
-        super().__init__(model=None, clip_values=None)
+        super().__init__(model=None, clip_values=None, channels_first=None)
         self._generator = generator
         self._discriminator_classifier = discriminator
         self._generator_loss = generator_loss
         self._generator_optimizer_fct = generator_optimizer_fct
         self._discriminator_loss = discriminator_loss
         self._discriminator_optimizer_fct = discriminator_optimizer_fct
 
-    def predict(self, x: np.ndarray, **kwargs) -> Any:  # lgtm [py/inheritance/incorrect-overridden-signature]
+    def predict(self, x: np.ndarray, batch_size: int = 128, **kwargs) -> np.ndarray:
```

Comment:
I was about to ask why is the batch_size parameter added when it's not used within the method but I can see neither is the existing **kwargs :-). Is that because you're trying to keep the method signatures consistent across estimators?

Reply:
Yes, if possible we try to have the same signatures for fit and predict methods and here I noticed it is possible.

Reply:
a ok yes that's perfect ! :-)